src/Task/YamlLint.php

```diff
@@ -56,12 +59,20 @@ public function canRunInContext(ContextInterface $context)
      */
     public function run(ContextInterface $context)
     {
-        $files = $context->getFiles()->name('/\.(yaml|yml)$/i');
+        /** @var array $config */
+        $config = $this->getConfiguration();
+        $whitelistPatterns = isset($config['whitelist_patterns']) ? $config['whitelist_patterns'] : [];
```

Comment:
```php
$whitelistPatterns = $config['whitelist_patterns'];
```

isset check is done in `getConfigurableOptions()`

Reply:
Hi, thanks for the reply.
I've deleted the isset check on $whitelistPatterns